install/10_cluster_reader_rbac.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-autoscaler-operator:cluster-reader
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
```

Comment:
is there any docs we can refer here for this label?

Reply:
Under already created aggregation rule this ClusterRole will be combined with `cluster-reader` list of permissions, according to https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles